mregsite/settings.py

```diff
@@ -167,6 +167,14 @@
     'DEFAULT_PERMISSION_CLASSES': (
         'mreg.api.permissions.IsInRequiredGroup',
     ),
+    'DEFAULT_THROTTLE_CLASSES': (
+    'rest_framework.throttling.AnonRateThrottle',
+    'rest_framework.throttling.UserRateThrottle'
+    ),
+    'DEFAULT_THROTTLE_RATES': {
+        'anon': '20/hour',
+        'user': '100/day'
```

Comment:
100/day seems very low. Actually almost a fifth of what an anonymous user can do.

Reply:
What is a good number?  1000/hour?  100/minute  As a API user, I would like the time period to be short, to be able to quickly recover if a script run amok.  It would be horrible to have to wait an entire day before continuing work, after messing up like that.